src/cpp/ext/filters/census/grpc_plugin.cc

```diff
@@ -57,6 +57,7 @@ void RegisterOpenCensusPlugin() {
 
 ::opencensus::trace::Span GetSpanFromServerContext(
     grpc::ServerContext* context) {
+  if (context == nullptr) opencensus::trace::Span::BlankSpan();
```

Comment:
Maybe I read this wrong, but this does not change context, hence it remains a nullptr and the nullptr deref in the next line would still crash?

Reply:
Missing return statement, good catch!